pkg/libsocketcan/Makefile

```diff
@@ -7,5 +7,8 @@ include $(RIOTBASE)/pkg/pkg.mk
 
 CFLAGS += -Wno-cast-align
 
+# HACK: Ignore `posix_headers` module here
```

Comment:
Minor nit. Consider expanding on this comment to explain that these are the POSIX headers provided by RIOT and not some system headers, and why we want this.

Reply:
Fixed, thx :)